src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let charge = 0;
+  const smallCharge = 20;
+  const bigCharge = 50;
+
+  if (days >= 3 && days < 7) {
+    charge = smallCharge;
+  }
+
+  if (days >= 7) {
+    charge = bigCharge;
+  }
+
+  const cost = days * 40 - charge;
+
+  return cost;
```

Comment:
```suggestion
  return days * 40 - charge;
```
And move 40 to the variable too